container-builder.spec.rpkg

```diff
@@ -0,0 +1,296 @@
+# For automatic rebuilds in COPR
```

Comment:
Can you please elaborate why adding this?

Reply:
Was a configuration used in the podman repo, removed